phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/app/ctrl/AppHomeController.kt

```diff
@@ -26,21 +25,27 @@ class AppHomeController(
     loggerFactory,
     firstModel = Home.emptyModel
 ) {
+    private var normalChannelsReady = false
 
     init {
+        launch {
```

Comment:
could we use the [`combine`](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/combine.html) method to watch both `bootChannelsFlow` and `channelsFlow` and have only one collector? That way I think we could remove the `normalChannelsReady` control.

Reply:
Great idea !

Reply:
Done in 00702cc